meshroom/ui/qml/Viewer3D/MediaLibrary.qml

```diff
@@ -277,6 +277,10 @@ Entity {
 
                 onCurrentSourceChanged: {
                     updateCacheAndModel(false)
+
+                    // Avoid the bounding box to disappear when we move it after a mesh already computed
+                    if(instantiatedEntity.hasBoundingBox && !currentSource)
+                        model.visible = true
```

Comment:
Maybe easier to check if there is a bounding box in the only place where the model can be set to `visible=false`.

I have not checked in detail but could be there:
From:
```
                onFinalSourceChanged: {
                    // update media visibility
                    // (useful if media was explicitly unloaded or hidden but loaded back from cache)
                    model.visible = model.requested;
```
To
```
                    model.visible = model.requested || instantiatedEntity.hasBoundingBox;
```